src/app/parcels/fetchParcelTableData.ts

```diff
@@ -190,9 +230,14 @@ export const getParcelsByIds = async (
     }
 
     const congestionCharge = await getCongestionChargeDetailsForParcels(data, supabase);
-    const formattedData = processingDataToParcelsTableData(data, congestionCharge);
+    const { parcelTableRows, error: processParcelDataError } =
+        await processingDataToParcelsTableData(data, congestionCharge);
+
+    if (processParcelDataError) {
+        throw new Error("Failed to process parcels.", { cause: processParcelDataError });
```

Comment:
I've had to put move `processingDataToParcelsTableData`'s throw here to keep the current behaviour without a big conflict with Amelia 😂